content/chainguard/libraries/faq.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Chainguard Libraries FAQ"
+linktitle: "FAQ"
+type: "article"
+description: "Frequently Asked Questions and answers for Chainguard Libraries users"
+draft: false
+menu:
+  docs:
+    parent: "libraries"
+weight: 003
+toc: true
+---
+
+
+## Frequently Asked Questions
+
+### Does Chainguard Libraries for Java include CVE remediation fixes?
```

Comment:
I would make these H2s

Reply:
Done .. more questions coming in future PR btw.